File: C_Programming/Assignment/Assignment3/factorial.c
#include<stdio.h>
void main()
{
    int num,fact=1;
    printf("Enter the value of no.:");
    scanf("%d",&num);
    int temp=num;
    while(num)
    {
        fact=fact*num;
        num--;

    }
    printf("\nfactorial of no:%d is %d",temp,fact);
}

//output:Enter the value of no.:5
//factorial of no:5 is 120

File: Java_Programs/Assignment3/Factorial.java
package Assignment3;

public class Factorial {

	public static void main(String[] args) {
		int num=5;
		int fact=1;
//	    printf("Enter the value of no.:");
//	    scanf("%d",&num);
	    int temp=num;
	    while(num==0)
	    {
	        fact=fact*num;
	        num--;

	    }
	    System.out.printf("\nfactorial of no:%d is %d",temp,fact);

	}

}
